config/internal/manager/rbac/agg-role.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: agg-manager-role
```

Comment:
Can you please tell me why we need this aggregationRule? Is this a new feature or it is related to this PR? I don't really see the benefits to aggregate the rules of the elevator to the manager, will the rule of the manager be cleared once the elevator is deleted?

Reply:
> Is this a new feature

what do you mean? It's in k8s for quite a while.

The biggest benefit of `ClusterRoleAggregation` over manually creating role binding for every role is simplicity. 

The elevator resource generation is decoupled from a specific service account the manager uses. Furthermore, I'd prefer this approach of having only a single controller managing certain webhook types (i.g. manager owns DerivedCR webhook) instead of splitting this responsibility in multiple controllers (e.g. manager having one and elevator having one webhook for the DerivedCR)

Reply:
I was thinking that the manager had the permission to handle tenantCRD instances. Sorry for that :)